spec/requests/extra_registrations_spec.rb

```diff
@@ -37,17 +37,17 @@
       end
 
       it 'Updates user name' do
-        expect { subject }.to change { user.reload.first_name }.to(user_params[:first_name])
+        expect { subject(:user) }.to change { user.reload.first_name }.to(user_params[:first_name])
```

Comment:
@tunde-anddigital my preference is to define a named subject block at the very top, then use the variable name.

Reply:
Yes - I agree

Reply:
@tunde-anddigital can you mark the PR as ready once this change and the conflicts are resolved please